erp/migrations/0114_alter_accessibilite_accueil_equipements_malentendants.py

```diff
@@ -0,0 +1,40 @@
+# Generated by Django 3.2 on 2021-04-15 11:21
+
+import django.contrib.postgres.fields
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("erp", "0113_remove_accessibilite_accueil_prestations"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="accessibilite",
+            name="accueil_equipements_malentendants",
+            field=django.contrib.postgres.fields.ArrayField(
+                base_field=models.CharField(
+                    blank=True,
+                    choices=[
+                        ("bim", "Boucle à induction magnétique fixe"),
+                        ("bmp", "Boucle à induction magnétique portative"),
+                        ("lsf", "Langue des signes française"),
+                        ("lpc", "Langue Française Parlée Complétée (LFPC)"),
+                        ("sts", "Sous-Titrage ou Transcription Simultanée"),
+                        ("autres", "Autres"),
+                    ],
+                    max_length=255,
+                ),
+                blank=True,
+                default=list,
+                null=True,
+                size=None,
+                verbose_name="Équipement(s) sourd/malentendant",
+            ),
+        ),
+        # didn't find a way to update an array easily without replacing the whole array with Django, so raw sql
+        migrations.RunSQL(
+            'update "erp_accessibilite" set "accueil_equipements_malentendants" = array_replace("accueil_equipements_malentendants", \'scd\', \'sts\')'
```

Comment:
Bah scratch my previous comment I didn't see that (I'm on mobile, gh ux is crap)